packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -122,6 +135,24 @@ class HookshotContent extends React.Component {
     }
   }
 
+  enableContentResizeListener() {
+    if (!this.contentResizeListenerAdded) {
```

Comment:
Would you be able to check for the presence of `this.resizeObserver` here and set it to null in disable to remove the additional booleans?

Reply:
I could, but I'd like to match the other listeners for this.